### ISSUE TITLE
Delete synapse disk files when app closes

### ISSUE THREAD

**Issue by afwillia** [Sage-Bionetworks/data_curator#278]
**Is your feature request related to a problem? Please describe.**
The app stores Synapse tables to the hard disk during each session. This information is redundant across sessions and different users and eventually fills up a lot of disk space. Since the stored information isn't used across sessions, it would be better to delete everything when the app closes.

**How important is this feature?** Select from the options below:
• 🏝 Low - it's an enhancement but not crucial for work

**When will use cases depending on this become relevant?** Select from the options below:
• Mid-term - 2-4 months

**Comment by allaway** [comment]
More on this here: https://github.com/Sage-Bionetworks/data_curator/issues/165

**Comment by milen-sage** [comment]
Add snippet to delete caches every time user logins to the app, as we discussed in CR. This will be tested in the shinyapps.io

**Comment by allaway** [comment]
FYI: shinyapps.io storage is ephemeral. When the app sleeps or is redeployed, stored data is purged. https://docs.rstudio.com/shinyapps.io/Storage.html

So, this might not be an issue in shinyapps.io.

**Comment by milen-sage** [comment]
Yep, we were discussing this - it will make the issue less poignant, however depending on deployment schedule and # users and app usage, it  sounds like may still create a problem.

**Comment by afwillia** [comment]
@ychae we are going to shelve this because it won't be an issue with hosting on shinyapps.io. However, it may come up when we switch over to AWS.

**Comment by milen-sage** [comment]
Closing for now and we can reopen/open a new issue as needed later.
